examples/bundle/pom.xml

```diff
@@ -395,7 +395,7 @@
         <dependency>
             <groupId>com.adobe.commerce.cif</groupId>
             <artifactId>core-cif-components-core</artifactId>
-            <version>${project.version}</version>
+            <version>1.1.0-SNAPSHOT</version>
```

Comment:
I merged https://github.com/adobe/aem-core-cif-components/pull/307 so you can revert that change. Sorry for the late fix.